pages/product/index.vue

```diff
@@ -19,79 +49,113 @@
-              <SfRating :score="0" :max="5" />
-              <a v-if="!!product.totalReviews" href="#" class="product__count">
-                ({{ product.totalReviews }})
-              </a>
-            </div>
+
+          <div class="product__rating">
+            <SfRating :score="3" :max="5" />
+            <a v-if="!!product.totalReviews" href="#" class="product__count">
+              ({{ product.totalReviews }})
+            </a>
+
             <SfButton class="sf-button--text" data-testid="read-all-reviews" @click="changeTab(2)">
-              Read all reviews
+              Write a Review
             </SfButton>
           </div>
         </div>
-        <div>
-          <p class="product__description desktop-only" v-html="description"></p>
-          <SfButton
-            data-cy="product-btn_size-guide"
-            class="sf-button--text desktop-only product__guide"
-          >
-            {{ $t("Size guide") }}
-          </SfButton>
-          <SfSelect
-            v-for="option in options"
-            :key="option.attributeFQN"
-            data-cy="product-select_size"
-            :value="productGetters.getOptionSelectedValue(option)"
-            :label="productGetters.getOptionName(option)"
-            class="sf-select--underlined product__select-size"
-            :required="option.isRequired"
-            @input="(value) => selectOption({ value }, option)"
-          >
-            <SfSelectOption
-              v-for="optionVal in option.values"
-              :key="optionVal.value"
-              :value="optionVal.value"
+
+        <div class="product__content">
+          <div class="product__description desktop-only" v-html="description"></div>
+          <div class="product__colors desktop-only">
+            <div class="product__color-label">{{ $t("Color") }}:</div>
+            <SfColor
+              v-for="(obj, i) in options.find((obj) => obj.attributeFQN.includes('color')).values"
+              :key="i"
+              data-cy="product-color_update"
+              :color="obj.value"
+              @click="
+                selectOption(
+                  { value: obj.value },
+                  { attributeFQN: 'Tenant~color' },
+                  'sf-color--active',
+                  $event
+                )
+              "
+            />
+          </div>
+
+          <div class="product__size">
+            <div
+              v-for="obj in options.find((obj) => obj.attributeFQN.includes('size')).values"
+              :key="obj.value"
+              class="sf-badge"
+              @click="
+                selectOption(
+                  { value: obj.value },
+                  { attributeFQN: 'Tenant~size' },
+                  'sf-badge--active',
+                  $event
+                )
+              "
+            >
+              {{ obj.value }}
+            </div>
+          </div>
+
+          <SfDivider class="divider-first" />
+          <SfRadio
```

Comment:
We will need to make these Shipping / Pickup radio buttons into their own components later

Reply:
Agreed